BUILDING.md

```diff
@@ -0,0 +1,79 @@
+
+# Building CircuitPython
+
+Welcome to CircuitPython!  
+
+This document is a quick-start guide only.
+
+Detailed guides on how to build CircuitPython can be found in the Adafruit Learn system at
+https://learn.adafruit.com/building-circuitpython/
+
+## Setup
+
+Please ensure you setup your build environment appropriately, as per the guide.  You will need:
+
+* Linux: https://learn.adafruit.com/building-circuitpython/linux
+* MacOS: https://learn.adafruit.com/building-circuitpython/macos
+* Windows Subsystem for Linux (WSL): https://learn.adafruit.com/building-circuitpython/windows-subsystem-for-linux
+
+### Submodules
+
+This project has a bunch of git submodules.  You will need to update them regularly.
+
+    git submodule sync
+    git submodule update --init --recursive
+
+### mpy-cross
+
+As part of the build process, mpy-cross is needed to compile .py files into .mpy files.
+To compile (or recompile) mpy-cross:
+
+    make -C mpy-cross 
+
+# Building
+
+There a number of ports of CircuitPython!  To build for your board, change to the appropriate ports directory and build.
+
+Examples:
+
+    cd ports/atmel-samd
+    make BOARD=circuitplayground_express
+
+    cd ports/nrf
+    make BOARD=circuitplayground_bluefruit
```

Comment:
May want to mention `-j` here for parallelizing the build. (I use `-j17` on a MacBook Pro.)

Reply:
I thought about that :-)  I resisted.  But since I'm not alone.